### PR TITLE
(DOCSP-8933): Port - Deployment Models & Regions doc

### PR DESCRIPTION
**Pull request by mongomoe**
JIRA:
-----
* https://jira.mongodb.org/browse/DOCSP-8933

Old/Adjacent Stitch Doc:
-------------------------
* https://docs.mongodb.com/stitch/admin/deployment-models-and-regions/

Stage:
-------
* https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-8933-deployment-models-and-regions-PORT/admin/deployment-models-and-regions.html

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -79,6 +79,8 @@ MongoDB Realm
    Deploy from the Realm UI </deploy/deploy-ui>
    Deploy Automatically with GitHub </deploy/deploy-automatically-with-github>
    Deploy with the Realm CLI </deploy/deploy-cli>
+   Deploy Automatically with GitHub </deploy/deploy-automatically-with-github>
+   Export an Existing Realm Application </deploy/export-realm-app>
```
Comment (nathan-contino-mongo) [inline source/index.txt]: You seem to have made some changes to some other parts of the doc here

File: source/deploy/export-realm-app.txt [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+</deploy/application-schema>` for more information on the schema
+and contents of Realm applications.
+
+.. include:: /includes/realm-application-file-schema.rst
+
+Summary
+-------
+
+- You can use Realm's export functionality to share your codebase, make a backup, or work with a version control system.
+- You can export Realm applications with the Realm UI or with the ``realm-cli`` :ref:`export <realm-cli-export>` command.
```
Comment (nathan-contino-mongo) [inline source/deploy/export-realm-app.txt]: I don't think this file should be in this PR.

Comment (mongomoe) [reply source/deploy/export-realm-app.txt]: removed

File: source/index.txt (hunk 1 above)
Comment (mongomoe) [reply source/index.txt]: fixed

**Pull request merged**